worker-test/pom.xml

```diff
@@ -340,6 +340,7 @@
                                 <ports>
                                     <port>${worker.testadminport2}:8081</port>
                                     <port>${worker.testdebugport2}:5005</port>
+                                    <port>8999:8080</port>
```

Comment:
We shouldn't use fixed ports in the build as they will lead to conflicts on the build machine is the port is already in use.